apps/web/app/(app)/onboarding/components/inapp/InviteTeamMate.tsx

```diff
@@ -0,0 +1,124 @@
+"use client";
+
+import OnboardingTitle from "@/app/(app)/onboarding/components/OnboardingTitle";
+import { ArrowRight } from "lucide-react";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
+import { toast } from "react-hot-toast";
+
+import { TTeam } from "@formbricks/types/teams";
+import { Button } from "@formbricks/ui/Button";
+import { Input } from "@formbricks/ui/Input";
+
+import { finishOnboardingAction, inviteTeamMateAction } from "../../actions";
+
+interface InviteTeamMateProps {
+  team: TTeam;
+  environmentId: string;
+  SET_CURRENT_STEP: (currentStep: number) => void;
+}
+
+const DEFAULT_INVITE_MESSAGE =
+  "I'm looking into Formbricks to run targeted surveys. Can you help me set it up? 🙏";
+const INITIAL_FORM_STATE = { email: "", inviteMessage: DEFAULT_INVITE_MESSAGE };
+
+function isValidEmail(email) {
+  const regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+  return regex.test(email);
+}
+
+function InviteMessageInput({ value, onChange }) {
+  return (
+    <textarea
+      rows={5}
+      placeholder="engineering@acme.com"
+      className="focus:border-brand-dark flex w-full rounded-md border border-slate-300 bg-transparent bg-white px-3 py-2 text-sm text-slate-800 placeholder:text-slate-400 focus:outline-none disabled:cursor-not-allowed disabled:opacity-50 dark:border-slate-500 dark:text-slate-300"
+      value={value}
+      onChange={onChange}
+    />
+  );
+}
+
+export function InviteTeamMate({ team, environmentId, SET_CURRENT_STEP }: InviteTeamMateProps) {
+  const [formState, setFormState] = useState(INITIAL_FORM_STATE);
+  const [isLoading, setIsLoading] = useState(false);
+  const router = useRouter();
+
+  const handleInputChange = (e, name) => {
+    const value = e.target.value;
+    setFormState({ ...formState, [name]: value });
+  };
+
+  const handleInvite = async () => {
+    if (!isValidEmail(formState.email)) {
+      toast.error("Invalid Email");
```

Comment:
<img width="731" alt="Screenshot 2024-02-27 at 18 13 17" src="https://github.com/formbricks/formbricks/assets/675065/110fc8c9-d2d2-41e0-bb86-531590d28fc9">

I'm getting an invalid email error, even though the email is valid :-)

Reply:
Modified the regex :)